Classes/Domain/Anomaly/RefindexUpdater.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace In2code\In2publishCore\Domain\Anomaly;
+
+/***************************************************************
+ *  Copyright notice
+ *
+ *  (c) 2015 in2code.de
+ *  Holger Krämer <post@holgerkraemer.com>
+ *
+ *  All rights reserved
+ *
+ *  This script is part of the TYPO3 project. The TYPO3 project is
+ *  free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  The GNU General Public License can be found at
+ *  http://www.gnu.org/copyleft/gpl.html.
+ *
+ *  This script is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  This copyright notice MUST APPEAR in all copies of the script!
+ ***************************************************************/
+
+use In2code\In2publishCore\Domain\Model\Record;
+use In2code\In2publishCore\Domain\Model\Task\RefindexUpdaterTask;
+use TYPO3\CMS\Core\SingletonInterface;
+
+/**
+ * Class RefindexUpdater
+ */
+class RefindexUpdater implements SingletonInterface
+{
+
+    /**
+     * @var \In2code\In2publishCore\Domain\Repository\TaskRepository
+     * @inject
```

Comment:
Please do not use @inject anymore, since in2publish doesn't use ObjectManager for Anomaly instantiation anymore